DataProducts/inc/STMTestBeamHeaders.hh

```diff
@@ -0,0 +1,109 @@
+#ifndef DataProducts_STMTestBeamHeaders_hh
+#define DataProducts_STMTestBeamHeaders_hh
+//
+// Header definitions for STM test beam data
+//
+
+// C++ includes
+#include <iostream>
+#include <vector>
+#include <array>
+#include <Rtypes.h>
```

Comment:
What in this file needs Rtypes?

Reply:
Must have come along for the ride when I copied another file. I've deleted it in this commit a19559e